doc/user_guide/statements/insert.md

```diff
@@ -0,0 +1,68 @@
+# INSERT 
+
+You can construct [`INSERT`](https://docs.exasol.com/sql/insert.htm) SQL statements using the `Insert` class.
+
+## Creating `INSERT` Statements
+
+Create an `INSERT` statement:
+
+```java
+        final Insert insert = StatementFactory.getInstance()
+                .insertInto("tableName")
+                .values("value1","value2","value3");
+```
+
+Create an `INSERT` statement for specific fields:
+
+```java
+        final Insert insert = StatementFactory.getInstance()
+                .insertInto("tableName")
+                .field("column1", "column2","column3")
+                .values("value1","value2","value3");
+```
+
+### Using Placeholders
+
+In SQL you can use value placeholders (`?`) in prepared statements. This allows you to add the values later in a safe way.
+
+you can add single placeholders like this:
+```java
+		        final Insert insert = StatementFactory.getInstance()
+                .insertInto("testTable")
+                .field("column1")
+                .valuePlaceholder();
+
+```
```

Comment:
Please, always add an empty line before and after a code block

Reply:
Done